.eslintignore

```diff
@@ -8,3 +8,5 @@ tmp/**
 docs/**
 _book/**
 website/**
+/packages/aws-client/*.js
```

Comment:
Why do we need this for `aws-client` if we didn't for `errors`? If we do need it, can we have `aws-client/.eslintignore`?

Reply:
This wasn't needed in the `errors` package because that package writes its compiled `.js` to a `dist` directory, which is covered by this top-level `.eslintignore` rule: `**/dist/**`.  In order to not break backward compatibility, we need the `aws-client` compiled Javascript files to appear at the root level of the `aws-client` package.

eslint does not support per-directory `.eslintignore` files. From the docs:

> When ESLint is run, it looks in the current working directory to find an .eslintignore file before determining which files to lint. If this file is found, then those preferences are applied when traversing directories. Only one .eslintignore file can be used at a time, so .eslintignore files other than the one in the current working directory will not be used.

I tried creating `aws-client/.eslintignore` and confirmed that it was not used. It looks like we have `.eslintignore` files in other places, too, which are also not going to have any effect. Well, they would only have an effect if you ran `eslint` from inside of that package directory.

Reply:
I have removed those extra `.eslintignore` files to prevent confusion in the future, and confirmed that removing them didn't change the behavior of `npm run eslint`.